### ISSUE TITLE
Preprocessing Dataset?

### ISSUE THREAD

**Issue by vasusingla** [erbloo/dr_cvpr20#1]
Hi, I was trying to replicate the results for the paper and had a question. The pre-trained models downloaded via Pytorch need to be normalized with Imagenet mean and standard deviation before being fed in, however, I did not see the images being normalized before being fed into the network in any of the scripts?

**Comment by erbloo** [comment]
Hello,
Due to torchvision.models docs, we do normalized images with imagenet mean and std for segmentations. Please refer to https://github.com/erbloo/bbox_std/blob/master/script_evaluate_segmentation_gt_pytorch.py as shown in readme.

**Comment by vasusingla** [comment]
I see, thanks for pointing this out. I believe this could be an issue for results on imagenet classification results. I could not see the normalization for images in the example notebook (https://github.com/erbloo/dr_cvpr20/blob/master/example.ipynb) or for the script for classification evaluation (https://github.com/erbloo/bbox_std/blob/master/script_evaluate_classification_pytorch.py). Thanks for the great paper btw!

**Comment by erbloo** [comment]
Thank you for the feedback! Actually we deprecated the classification related code in current bbox_std repo, since the classification-to-classification transferability problem is not discussed in the paper. I'll update the https://github.com/erbloo/bbox_std repo ASAP. Hope our work could help you on your research :)
